bitwarden/__init__.py

```diff
@@ -20,86 +20,35 @@ class Plugin(PluginInstance, TriggerQueryHandler):
 
     def __init__(self):
         TriggerQueryHandler.__init__(self,
-                                     id=md_id,
+                                     id=md_iid,
```

Comment:
dont use md_iid here. md_id is the id of the plugin, md_iid is the interface id, currently[ v2.1](https://raw.githubusercontent.com/albertlauncher/python/master/albert.pyi)

Reply:
Sorry, I thought it was a misspelling because I didn't find where 'md_id' was declared, the linter marked as not defined